tools/data/llama2_asym.csv

```diff
@@ -0,0 +1,11 @@
+"int4_asym, gs64",mode,%int4,%int8,lora rank,plot name,"model size, Gb","wikitext, word perplexity","lambada-openai, acc","lambada-openai, perplexity","WWB, similarity",compress_time_in_min
```

Comment:
@ljaljushkin I could not agree with publishing the pivot table in the repository. Please think about having one place of the data.

Reply:
discussed with @alexsu52 offline. 
Agreed to keep one pivot table for checking maintenance of the script and to showcase the usage.

Reply:
Done